docs/guide/http_cache.md

```diff
@@ -267,6 +266,42 @@ ezpublish:
     instructions on how to generate a Fastly API token.
     The token needs `purge_select` and `purge_all` scope.
 
+    #### Ensure proper Captcha behavior
+
+    If your installation uses Varnish and you want users to be able to configure and use Captcha in their forms, you must enable the sending of Captcha data as a response to an Ajax request.
```

Comment:
@MagdalenaZuba Thanks for the merge. 

As for the change, I would guess the same here is true for Fastly? Or even also Symfony Proxy? ...Which issues does this come from (further details on the issue).

Reply:
It comes from 2.5.9 release PR: https://github.com/ezsystems/developer-documentation/pull/1083